python/deltalake/writer.py

```diff
@@ -44,6 +44,10 @@ def write_deltalake(
     partition_by: Optional[List[str]] = None,
     filesystem: Optional[pa_fs.FileSystem] = None,
     mode: Literal["error", "append", "overwrite", "ignore"] = "error",
+    file_options: Optional[ds.FileWriteOptions] = None,
```

Comment:
I think we can make this parquet-specific, since we only support parquet:

```suggestion
    file_options: Optional[ds.ParquetFileWriteOptions] = None,
```